src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.cs

```diff
@@ -23,6 +23,41 @@ public class RaspberryPi3Driver : GpioDriver
         private readonly Set_Register _setClearRegister;
         private readonly Get_Register _getClearRegister;
 
+        /// <summary>
+        /// Default Gpio mode. Use <see cref="SetPinMode"/> to switch between input and output.
+        /// </summary>
+        public static readonly ExtendedPinMode GpioMode = new RaspberryPiExtendedPinMode("GPIO", 0);

```

Comment:
- make them public rather than protected

Reply:
@krwq, @joperezr Updated PR according to our discussion. I did have to add a property to query the support, because it doesn't currently work on Windows, and I don't necessarily want to rely on exceptions in this case. (And Windows 10 IOT support on the Raspi is dying out it seems, anyway)